Added save utils. (#26)
szczurek-lab/lso: lso/model/README.md

in this case we do not pass any params to `get_instance`) and other for which we provide previously obtained `instance_params`
(think e.g., about model deserialization when one loads params from memory and passes them to a `Model`).

> ### Note on serialization
> 
> We assume that a model might serialized to `JSON` so it has a method `get_config_dict`. It should be also able to be deserialized
> from JSON, so it has a `classmethod` `from_config_dict`.
> 

Reviewer: ```suggestion
### Note on serialization

Each `Model` serializes to `JSON` with `get_config_dict` method and deserializes
from JSON using `from_config_dict` `classmethod`.

```
